src/webgl/material.js

```diff
@@ -300,23 +300,32 @@ p5.prototype.createFilterShader = function (fragSrc) {
     }
   `;
   let defaultVertV2 = `#version 300 es
-    uniform mat4 uModelViewMatrix;
-    uniform mat4 uProjectionMatrix;
+  uniform mat4 uModelViewMatrix;
+  uniform mat4 uProjectionMatrix;
 
-    in vec3 aPosition;
-    in vec2 aTexCoord;
-    out vec2 vTexCoord;
+  in vec3 aPosition;
+  in vec2 aTexCoord;
+  in vec3 aNormal;
 
-    void main() {
-      // transferring texcoords for the frag shader
-      vTexCoord = aTexCoord;
+  out vec3 faNormal;
+  out vec3 vNormal;
+  out vec3 faPosition;
+  out vec2 vTexCoord;
+  out vec3 fvNormal;
```

Comment:
I think we don't use these in the shader, we can probably take them out from here